app/models/product_drive.rb

```diff
@@ -62,4 +62,14 @@ def self.search_date_range(dates)
     dates = dates.split(" - ")
     @search_date_range = { start_date: dates[0], end_date: dates[1] }
   end
+
+  def items_quantity_by_name
```

Comment:
This should really be `item_quantities_by_name`

Reply:
@dorner thanks, this is done.